types/three/examples/jsm/transpiler/TSLEncoder.d.ts

```diff
@@ -1,7 +1,30 @@
-import { Program } from "./AST.js";
+import { Program, Statement } from "./AST.js";
 
 export default class TSLEncoder {
     constructor();
 
+    tab: string;
+    imports: Set<string>;
+    global: Set<string>;
+    overloadings: Map<string, string>;
+    iife: boolean;
+    uniqueNames: boolean;
+    reference: boolean;
+
+    addImport(name: string): void;
+    emitUniform(node: Statement): string;
```

Comment:
I'm still a little confused why we can't make these parameter types more specific. Doesn't this method assume the parameter is a `Uniform`, for example?